implementation.md

```diff
@@ -344,22 +402,24 @@ When queried by the user, an action can return a JSON representation. This is
 arbitrary and entirely for the benefit of users. Any information can be
 included in an appropriate way.
 
+##### Manual Implementation
+
```

Comment:
As before: tell me why I would need manual + move this to the end of this section.